src/test/java/com/waverleysoftware/testcases/CreateAccountTests.java

```diff
@@ -0,0 +1,38 @@
+package com.waverleysoftware.testcases;
+
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+public class CreateAccountTests  extends BaseTest {
```

Comment:
makes sure that you follow formatting rules... seems like a redundant whitespace